backend/controllers/productController.js

```
@@ -0,0 +1,187 @@
+      bestSeller,
+    } = req.body;
+
+    const image1 = req.files.image1?.[0];
+    const image2 = req.files.image2?.[0];
+    const image3 = req.files.image3?.[0];
+    const image4 = req.files.image4?.[0];
+
+    const images = [image1, image2, image3, image4].filter(
+      (item) => item !== undefined
+    );
+    let imagesUrl = images.map((item) => {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'imagesUrl' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)